UofTGroup3/toronto-newcomers-guide
# # This creates a dynamic dashboard page.

## Overview

>The goal of this pull request is to enhance the user experience for newcomers to the GTA by providing personalized recommendations and links on their dashboard. These recommendations are dynamically generated based on the user's selected profile type.

>This pull request aims to implement personalized recommendations and links on the user dashboard for newcomers to the GTA. By adding this feature, we provide valuable, profile-specific information that can aid in a smoother transition and quicker adaptation to the new environment.

## Changes Made

Added a new component that fetches user data based on the selected profile type.
Used local storage to store and receive data.
Defined functions and event listener to create and display recommendations dynamically based on the selected profile type.
Styled by using Materialize CSS library and the Custome CSS.

## How to Test
Log in to the application.
Navigate to the dashboard.
Verify that recommendations and links are personalized based on the selected profile type.
Run unit tests to confirm that the feature meets the acceptance criteria.
